tests/manual-test-cases/Group19-ROBO/19-2-ROBO-Container-Limit.md

```diff
@@ -0,0 +1,50 @@
+Test 19-2 - ROBO - Container VM Limit
+=======
+
+# Purpose:
+To verify that the total container VM limit feature works as expected in a vSphere ROBO Advanced environment.
+
+# References:
+1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+2. [Limit total allowed containerVMs per VCH](https://github.com/vmware/vic/issues/7273)
+3. [vic-machine inspect to report configured containerVM limit](https://github.com/vmware/vic/issues/7284)
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
+* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
+* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+
+See https://confluence.eng.vmware.com/display/CNA/VIC+ROBO for more details.
+
+# Test Steps:
+1. Deploy a ROBO Advanced vCenter testbed for both environments above
+2. Install the VIC appliance on vCenter with a container VM limit of y
```

Comment:
Which cluster are we installing to? Is it random, round robin, each in turn?

Reply:
We could pick any cluster to install into, unless we want to run this test in each cluster configuration - in which case I can mention so in the `Environment` section above.